dags/glam.py

```diff
@@ -15,6 +19,7 @@
     'retries': 1,
     'retry_delay': timedelta(minutes=30),
 }
+glam_bucket = "gs://glam-dev-bespoke-nonprod-dataops-mozgcp-net",
```

Comment:
This is invalid syntax.

```suggestion
glam_bucket = "gs://glam-dev-bespoke-nonprod-dataops-mozgcp-net"
```